### DISCUSSION TITLE
Changing the favicon?

### DISCUSSION THREAD

**Discussion by iYonga** [DirectoryLister/DirectoryLister#610] [Help & Support] [answered]
How to change the favicon? I tried uploading a favicon.ico to the root, but that didn't work.

I know I ask too much but this will provide ease for other people that are searching for this.

**Comment by PHLAK** [comment, marked as answer]
This should be possible by editing the `<link rel="icon" href="...">` element in `app/views/layouts/app.twig`. You will also need to clear the application cache by deleting the contents of `app/cache/views` to see the changes. You may also need to clear your browsers cache.
